src/services/algorithms/services/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
How about `src/services/algorithms/init/` with `algorithms_init.cc` and `AlgorithmsInit_service`? It would require the factory to have `Service<AlgorithmsInit_service> m_algorithmsInitSvc{this};`

Reply:
Hmm, won't work due to `get_filename_component(PLUGIN_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)` but I've always wanted to change that into the relative path under `src` with `/` changed to `_`.

Reply:
How about `src/services/algorithms_init/` with `algorithms_init.cc` and `AlgorithmsInit_service`? That doesn't put some generic `init.so` plugin in a common dir.

Reply:
Sounds good to me!

Reply:
Renamed in https://github.com/eic/EICrecon/pull/1195/commits/b91d559c6d57dea32b71b6808a7ca55e9bb0bb58 and https://github.com/eic/EICrecon/pull/1195/commits/07adf7ce7ed27d1ecc0bbf79b9c55c35c7caba30